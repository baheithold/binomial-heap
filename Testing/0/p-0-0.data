9 1 390 ; 4 4 491 ; 9 5 432 ; 9 6 51 ; 1 4 62 ; 

