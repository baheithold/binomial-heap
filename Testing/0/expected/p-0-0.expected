0: 7
1: 2(7)421 6(7)383
2: 9(2)386
3: 3(9)190
4: 5(3)415
weight: 1795
